<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1006</width>
    <height>743</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Events</string>
       </attribute>
       <widget class="QPushButton" name="pushButtonEvents">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>130</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Get events</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>10</y>
          <width>141</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>130</y>
          <width>131</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Football</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>IceHockey</string>
         </property>
        </item>
       </widget>
       <widget class="QPushButton" name="pushButtonFunds">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Get Funds</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="tableWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>170</y>
          <width>631</width>
          <height>461</height>
         </rect>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <column>
         <property name="text">
          <string>Name</string>
         </property>
        </column>
       </widget>
       <widget class="QLineEdit" name="lineEditFilter">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>100</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>100</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Filter:</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="tableWidgetWalletHistory">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>10</y>
          <width>511</width>
          <height>141</height>
         </rect>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>75</number>
        </attribute>
        <column>
         <property name="text">
          <string>Date</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>CashIn</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Betfair</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Ext</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Total</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Balance</string>
         </property>
        </column>
       </widget>
       <widget class="QLineEdit" name="lineEditBf">
        <property name="geometry">
         <rect>
          <x>820</x>
          <y>70</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonAddWalletEntry">
        <property name="geometry">
         <rect>
          <x>780</x>
          <y>130</y>
          <width>161</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditUB">
        <property name="geometry">
         <rect>
          <x>820</x>
          <y>100</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>780</x>
          <y>70</y>
          <width>31</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Betfair</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>780</x>
          <y>100</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Ext</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditCashIn">
        <property name="geometry">
         <rect>
          <x>820</x>
          <y>40</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>780</x>
          <y>40</y>
          <width>31</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>CashIn</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelBalance">
        <property name="geometry">
         <rect>
          <x>780</x>
          <y>10</y>
          <width>151</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Balance: </string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>170</y>
          <width>331</width>
          <height>461</height>
         </rect>
        </property>
        <property name="title">
         <string>SS Games</string>
        </property>
        <widget class="QTableWidget" name="tableWidgetExtSS">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>311</width>
           <height>431</height>
          </rect>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>300</number>
         </attribute>
         <attribute name="horizontalHeaderMinimumSectionSize">
          <number>37</number>
         </attribute>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
        </widget>
       </widget>
       <widget class="QPushButton" name="pushButtonHorn">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>50</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="text">
    <string>Login</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
